tests/data/graph/querybuilder/sample_data/case_insensitive_prop_ref.py

```diff
@@ -1,33 +1,34 @@
+FAKE_GITHUB_ORG_DATA = {
+    'url': 'https://example.com/my_org',
+    'login': 'my_org',
+}
+
 FAKE_GITHUB_USER_DATA = [
     {
+        'MEMBER_OF': FAKE_GITHUB_ORG_DATA['url'],
         'hasTwoFactorEnabled': None,
-        'node': {
-            'url': 'https://example.com/hjsimpson',
-            'login': 'HjsimPson',  # Upper and lowercase
-            'name': 'Homer Simpson',
-            'isSiteAdmin': False,
-            'email': 'hjsimpson@example.com',
-            'company': 'Springfield Nuclear Power Plant',
-        },
+        'url': 'https://example.com/hjsimpson',
```

Comment:
[Clarification] 
Double checking: the object here flattened out where that `node` is gone because it goes through some transformations now, right?

Reply:
Yeah.  The tests that use this data call the `load_users` function, which deals with transformed data, which has been flattened out.  It's a little different than the data used over in `test_users.py`, which tests the overall sync and mocks the 'raw' data that comes out of the get methods.  _That_ data has the `node` and some other stuff.  If there's anything else to comment on or dig into here, let me know.